### DISCUSSION TITLE
[Feature Request] Option to display field instructions as tooltip

### DISCUSSION THREAD

**Discussion by TheFunkyMonk** [craftcms/cms#8988] [Ideas]
### Description

The inline field instructions display for top-level fields can get a bit unwieldy for longer instructions, particularly when combined with the field layout editor to display fields side-by-side.

![Screen Shot 2021-01-14 at 11 21 26 AM](https://user-images.githubusercontent.com/1728770/104638193-b47bf500-565a-11eb-822a-1e027f3c2351.png)

### Suggestion

It would be nice to have an option when creating fields (or even a general config setting to always) display instructions behind the info icon/tooltip that shows up for nested fields, like so:

![Screen Shot 2021-01-14 at 11 20 41 AM](https://user-images.githubusercontent.com/1728770/104638219-bd6cc680-565a-11eb-8889-b9e6f8f0da48.png)

**Comment by piotrpog** [comment]
Long descriptions sometimes bloat entry page in control panel, putting it in tooltip would help a lot.

**Comment by missmatsuko** [comment]
In general, I think tooltips for field instructions may cause issues since content authors could miss important information.

I can see that instructions being redundant if there's multiple of the same blocks within a matrix field.

Would it meet your needs if we included an accessibility setting that puts instructions as tooltips in matrix blocks only?

As for the example in your screenshot specifically, perhaps it would be possible to reduce or remove the instructions text entirely by cropping the image to match the video's aspect ratio instead of asking the author to do so?

**Comment by TheFunkyMonk** [comment]
Thanks for the response! The actual instruction text in the screenshot above maybe isn't the best example content-wise, just the first example of a field with some instruction text looking visually off I noticed.

It's really more of a visual thing since having it in side-by-side layouts like the above can get kind of awkward, so for less-critical instructions, but those I'd still like to be able to include for author reference, it'd be nice to have the option to hide behind a tooltip or something less visually-prominent. I was thinking it would be nice as an optional display for the instructions in the field layout designer view where you can hide the label for a field. Maybe "Instructions" has its own checkbox to display as a tooltip, which is off by default.

![Screen Shot 2021-04-27 at 3 19 25 PM](https://user-images.githubusercontent.com/1728770/116319799-01985880-a76c-11eb-91a2-7ad45663b9eb.png)

**Comment by missmatsuko** [comment]
Hmm would it work for you if there were another option there just for supplementary information and that would show as a tooltip? 

Instructions sounds to me like pretty important information that shouldn't go in a tooltip.

**Comment by TheFunkyMonk** [comment]
That would totally work, too! And probably a better way to approach it. Just any way to include some extra information about a field without it being as visually prominent as instructions currently display would be great.
